tests/fe/templating_test.py

```diff
@@ -0,0 +1,29 @@
+import itertools
+import json
+import os
+from base64 import b64encode
+from hashlib import sha256
+
+import grouper.fe
+from grouper.fe.settings import FrontendSettings
+from grouper.fe.templating import FrontendTemplateEngine
+
+
+def test_included_resources():
+    # type: () -> None
+    settings = FrontendSettings()
+    static_path = os.path.join(os.path.dirname(grouper.fe.__file__), "static")
+    engine = FrontendTemplateEngine(settings, "", static_path, package="tests.fe")
```

Comment:
Are we going to have different templates in tests.fe than grouper.fe?

Reply:
Yup, that's where the `base.json.tmpl` template used in the test comes from. It's nonsensical for the production code, just a way to generate a template and then be able to easily inspect, from Python, what template variables were set.